sqlmypy.py

```diff
@@ -110,6 +111,53 @@ def add_model_init_hook(ctx: ClassDefContext) -> None:
     add_method(ctx, '__init__', [kw_arg], NoneTyp())
     ctx.cls.info.metadata.setdefault('sqlalchemy', {})['generated_init'] = True
 
+    for stmt in ctx.cls.defs.body:
+        if not (isinstance(stmt, AssignmentStmt) and len(stmt.lvalues) == 1 and isinstance(stmt.lvalues[0], NameExpr)):
+            continue
+
+        # We currently only handle setting __tablename__ as a class attribute, and not through a property.
+        if stmt.lvalues[0].name == "__tablename__" and isinstance(stmt.rvalue, StrExpr):
```

Comment:
Maybe we can also support constants like:
```python
NAME: Final = 'users'

class User(Base):
    __tablename__ = NAME
    ...
```
No need to this in this PR, but may worth leaving a TODO and/or a follow-up issue.